ajax/libs/cell/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cell",
+  "filename": "cell.min.js",
+  "description": "A self-driving web app framework",
+  "homepage": "https://www.celljs.org",
+  "keywords": [],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/intercellular/cell.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "cell.js",
+          "cell.min.js"
+        ]
```

Comment:
@devsnek I think I understand what you're trying to say 👍 

@PeterDaveHello Your opinion on this please.